app/controllers/registers_controller.rb

```diff
@@ -4,8 +4,9 @@ def new
   end
 
   def create
-    @user = User.new(user_params)
-    if @user.save
+    @user = User.create(user_params)
+    if @user.valid?
+      sign_in @user
```

Comment:
new
check valid
if valid
save
dùng callbacks sau khi save thì sign in em nhé

Reply:
cái này k nên dùng call back thì hơi thừa á @longnh-3165 . chỉ cần sau khi tạo thành công  sign_in luôn cần gì call back nhỉ. vì sau này lở adm có quyền tạo user thì sao =))

Reply:
ke a don